### DISCUSSION TITLE
GoPro 11 Camera Freezes and Reboots When Activating WIDE Mode (Firmware 2.20.70)

### DISCUSSION THREAD

**Discussion by Diadon81** [gopro/labs#696] [Q&A]
Hi,
Hello, whenever I try to activate WIDE mode on my GoPro 11 Black with firmware version 2.20.70, the camera immediately freezes and goes into a constant reboot loop. My QR code: mV2r4p25fW0te4aAcFd1w55i8M1S180sHg1dVoV0!MWIDE=1!MBITR=170!MNR01=1!MLOGB=400
. Has anyone encountered a similar issue and knows how to fix it?

**Comment by dnewman-gpsw** [comment]
Generally don't mix permanent !Mwxyz= commands with Protune settings, as you will not solve the issue. Start with a !RESET with required button presses. Add commands incrementally to see where the issue is.
